src/core/mount-setup.c

```diff
@@ -405,7 +406,15 @@ int mount_setup(bool loaded_policy) {
          * systemd. */
         mkdir_label("/run/systemd", 0755);
         mkdir_label("/run/systemd/system", 0755);
-        mkdir_label("/run/systemd/inaccessible", 0000);
+        mkdir_label(SD_INACCESSIBLE_PATH, 0755);
+
+        // Set up inaccessible items
+        mknod(SD_INACCESSIBLE_PATH_REG, S_IFREG | 0000, 0);
```

Comment:
> does this actually work? I wasn't aware you could use mknod () for files, too...

[Man page](http://man7.org/linux/man-pages/man2/mknod.2.html) specify `S_IFREG`, so I suppose it does.